app/services/folio/types.rb

```diff
@@ -86,6 +88,68 @@ def get_type(type)
       JSON.parse(file.read) if file.exist?
     end
 
+    # Mapping and functions for finding specific information
+    # For FOLIO, destination is specified as service point
+    # Convert service point to library for scheduling and library hours
+    def map_to_library(service_point_code)
```

Comment:
Can we change the name of this method to `map_to_library_code` since it returns a code and not a Library object?

Reply:
yes